types/react/test/tsx.tsx

```diff
@@ -357,6 +357,22 @@ class LegacyContext extends React.Component {
     }
 }
 
+class LegacyContextWithConstructor extends React.Component {
+    static contextTypes = { foo: PropTypes.node.isRequired };
+
+    constructor(props: {}, context: {}) {
+        super(props, context);
+    }
+
+    render() {
+        // $ExpectType unknown
+        this.context;
+        return (this.context as any).foo;
+    }
+}
+
+<LegacyContextWithConstructor />;
+
```

Comment:
Let's move this into `reactNodeTests` instead. It already has a group for "ReturnWithLegacyContext" so `RenderWithLegacyContext` fits right next to it.

Reply:
I’m guessing you actually meant `elementTypeTests`, so I moved the tests there.